### DISCUSSION TITLE
Worth to package it as deb?

### DISCUSSION THREAD

**Discussion by grossmogul** [taviso/123elf#49] [Ideas]
Thanks Taviso for that piece of history!

It built out of the box and I asked myself if it should get an own deb package. With that it would get easier to install w/o compiling 123 at least on Debian. I could assemble and provide the package.

What do you think?

Best regards
Theo

PS: I stumbled over it here: https://www.golem.de/news/retro-computing-lotus-1-2-3-auf-linux-portiert-2205-165676.html

**Comment by taviso** [comment]
I've made a first attempt, although I'm not a .deb expert!

https://github.com/taviso/123elf/releases/tag/v1.0.0-rc1

I would appreciate some testing.
